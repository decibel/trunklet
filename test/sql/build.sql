\set ECHO none
CREATE EXTENSION IF NOT EXISTS variant;

\i test/helpers/psql.sql

BEGIN;
\i sql/trunklet.sql
ROLLBACK;
