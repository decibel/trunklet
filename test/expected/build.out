\set ECHO none
 ?column? 
----------
 t
(1 row)

 ?column? 
----------
 t
(1 row)

 ?column? 
----------
 t
(1 row)

